libs/components/popovers/testing/src/popover/popover-fixture.ts

```diff
@@ -62,30 +62,30 @@ export class SkyPopoverFixture {
   }
 
   //#region helpers
-  private get contentElement(): HTMLElement {
-    return this.queryOverlay('sky-popover-content');
+  private get contentElement(): HTMLElement | undefined {
+    return this.queryOverlay('sky-popover-content') || undefined;
   }
 
-  private get containerElement(): HTMLElement {
-    return this.queryOverlay('.sky-popover-container');
+  private get containerElement(): HTMLElement | undefined {
+    return this.queryOverlay('.sky-popover-container') || undefined;
   }
 
-  private get titleElement(): HTMLElement {
-    return this.queryOverlay('.sky-popover-title');
+  private get titleElement(): HTMLElement | undefined {
+    return this.queryOverlay('.sky-popover-title') || undefined;
   }
 
-  private get bodyElement(): HTMLElement {
-    return this.queryOverlay('.sky-popover-body');
+  private get bodyElement(): HTMLElement | undefined {
+    return this.queryOverlay('.sky-popover-body') || undefined;
   }
 
-  private getOverlay(): HTMLElement {
-    return document.querySelector('sky-overlay');
+  private getOverlay(): Element | undefined {
```

Comment:
It seems like we should keep this consistent as `HTMLElement` or change other `HTMLElement` references to `Element`.

Reply:
This is a cherry-pick commit, so we'll need to fix it in `5.x.x` as well.

In any case, when querying against the `document`, TypeScript returns either `Element | null`, whereas when querying against an `HTMLElement` TypeScript returns `HTMLElement | null`. I figured we should respect what the native API returns instead of casting it as an extended type. (This is also a private method so it's not a breaking change.)